pkg/kubelet/cm/container_manager.go

```diff
@@ -52,6 +54,10 @@ type ContainerManager interface {
 
 	// GetNodeAllocatable returns the amount of compute resources that have to be reserved from scheduling.
 	GetNodeAllocatableReservation() v1.ResourceList
+
+	// UpdateQOSCgroups performs housekeeping updates to ensure that the top
+	// level QoS containers have their desired state in a thread-safe way
+	UpdateQOSCgroups() error
```

Comment:
i know @vishh asked that this be called "Containers".

similar to the discussion on node allocatable, i am not going to hold up this pr on naming conventions.

we can fix that across the board in a later pr.

right now "Cgroups" is consistent with "PodContainerManager" terminology.